docs/using-react-redux/selector-functions.md

```diff
@@ -0,0 +1,261 @@
+---
+id: selector-functions
+title: Selector Functions
+hide_title: true
+sidebar_label: Selector Functions
```

Comment:
I have noticed that this section is not appearing in the navigation on the left? How can I add it there?

Reply:
Hi, the sidebar configurations are at `website/sidebars.json`. You may check out the changes in [this PR](https://github.com/reduxjs/react-redux/pull/1042).

Reply:
Thanks Wei. I have added.